packages/react-reconciler/src/ReactFiber.old.js

```diff
@@ -529,6 +526,13 @@ export function createFiberFromTypeAndProps(
           return createFiberFromTracingMarker(pendingProps, mode, lanes, key);
         }
       // eslint-disable-next-line no-fallthrough
+      case REACT_DEBUG_TRACING_MODE_TYPE:
+        if (enableDebugTracing) {
+          fiberTag = Mode;
+          mode |= DebugTracingMode;
+          break;
+        }
+      // eslint-disable-next-line no-fallthrough
```

Comment:
The trick is to place these at the bottom so if the condition it false it's the same as `default` which means that the case is unnecessary which Closure is smart enough to understand but not Rollup.